README.md

```diff
@@ -1 +1,49 @@
-# pe-state-machine-ms
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>PE-JS   DIF Presentation Exchange JavaScript Library 
+  <br>
+</h1>
+
+[![CI](https://github.com/Sphereon-Opensource/pe-js/actions/workflows/main.yml/badge.svg)](https://github.com/Sphereon-Opensource/pe-js/actions/workflows/main.yml)
+
+## Active Development
+_IMPORTANT: This software still is in early development stage. As such you should expect breaking changes in APIs, we expect to keep that to a minimum though._
+
+## Background
+
+The pe-state-machine project aims to implement the PE-OPENAPI and use PE-JS library and Veramo API as a dependency. 
+
+## How to run it
+
+From the root folder:
```

Comment:
Perhaps we can add DB creation command for mongo

Reply:
There is no such thing. The database is created on the first insertion.